src/platform/STM32/mk/STM32F7.mk

```diff
@@ -193,7 +191,13 @@ MSC_SRC = \
             msc/usbd_storage_sd_spi.c
 
 SPEED_OPTIMISED_SRC += \
-            common/stm32/system.c
+            stm32/system.c \
+            bus_i2c_hal.c \
```

Comment:
(You can tell where is this file located?)

Reply:
this one is located under platform/STM32. Let me rethink this one.

Reply:
Done :)